app/server/index.jsx

```diff
@@ -98,21 +96,21 @@ app.get('*', function (req, res) {
     let status = state.error ? state.error : 200
 
     let componentHead = ReactDOMServer.renderToStaticMarkup(<Head {...state.app.pageData} error={state.app.error} />)
-    let componentScripts = ReactDOMServer.renderToStaticMarkup(<Scripts />)
-    let renderedHtml = renderFullPageHtml(componentHtml, componentHead, componentScripts, JSON.stringify(state))
+    let renderedHtml = renderFullPageHtml(componentHtml, componentHead, JSON.stringify(state))
```

Comment:
@enginehouse was this deliberate? It would be good to keep as much templating in React including things like scripts. Can you not do this in the `<Scripts />` component?

Reply:
It was, because the timestamp is being generated in `config-loader.js` and I figured I would keep it there as are the only js files we are using and are pretty constant - can easily shift them out though

Reply:
Ok - it was working - i.e. passing timestamp to the component, but let me double check it...

Reply:
I don't think there was any client side react in the version I had orginally, but like I say, I'll switch it back